MarsMission/ViewModels/WeatherDataViewModel.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  WeatherDataViewModel.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/04.
+//
+
+import Foundation
+
+
+class WeatherDataViewModel {
+	
+	let repository = WeatherDataRepositoryImplemantation()
+	
+	var weatherdata: WeatherData?
+	var view: WeatherView
+	
+	init(view: WeatherView) {
+		self.view = view
+	}
+	
+	func gettingWeatherData() {
+		DispatchQueue.global(qos: .background).async {
+			self.repository.getWeatherData { (result) in
+				switch result {
+					case .success(let weatherdata):
+						self.weatherfecthWeatherDataSuccess(weatherdata)
+					case .failure(let error):
+						self.weatherfecthWeatherDataFail(error)
+				}
+			}
+		}
+	}
+	
+	private func weatherfecthWeatherDataSuccess(_ weatherdata: WeatherData) {
```

Comment:
spelling of fetch

Reply:
<img width="538" alt="Screenshot 2020-11-06 at 15 59 48" src="https://user-images.githubusercontent.com/55865505/98374416-4f4bc800-2049-11eb-8fb9-492b921dcd73.png">

Reply:
<img width="524" alt="Screenshot 2020-11-06 at 16 07 32" src="https://user-images.githubusercontent.com/55865505/98375091-4f989300-204a-11eb-9df6-1a0d02ad056e.png">